src/EnergyPlus/DataSurfaces.cc

```diff
@@ -237,57 +237,6 @@ namespace DataSurfaces {
     int const InConvWinLoc_WindowBelowThis(4);         // this is a wall with window below it
     int const InConvWinLoc_LargePartOfExteriorWall(5); // this is a big window taking up most of wall
 
-    // Parameters for window shade status
-    int const NoShade(-1);
-    int const ShadeOff(0);
-    int const IntShadeOn(1); // Interior shade on
-    int const SwitchableGlazing(2);
-    int const ExtShadeOn(3);  // Exterior shade on
-    int const ExtScreenOn(4); // Exterior screen on
-    int const IntBlindOn(6);  // Interior blind on
-    int const ExtBlindOn(7);  // Exterior blind on
-    int const BGShadeOn(8);   // Between-glass shade on
-    int const BGBlindOn(9);   // Between-glass blind on
-    int const IntShadeConditionallyOff(10);
-    int const GlassConditionallyLightened(20);
-    int const ExtShadeConditionallyOff(30);
-    int const IntBlindConditionallyOff(60);
-    int const ExtBlindConditionallyOff(70);
-
-    // WindowShadingControl Shading Types
```

Comment:
@mjwitte Please see [this](https://github.com/NREL/EnergyPlus/pull/8435#commitcomment-46551726) thread for the explanation. You are right. Mixing the types and status was not an optimal design decision in the first place (I only reassured this when I was 80% done with this branch). But changing this may require some redesign of the logic and a lot more refactoring. It's more like a refactoring rather than a 10x effort. I think this could happen later.